### PR TITLE
flashback: add limitation with stale read/snapshot

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by YuJuncen — APPROVED** [review]

File: sql-statements/sql-statement-flashback-cluster.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ TiDB v6.5.6、v7.1.3、v7.5.1、v7.6.0 开始引入了 `FLASHBACK CLUSTER TO TSO
 > **警告：**
 >
 > 在指定恢复时间点时，请务必检查 TIMESTAMP 或 TSO 的有效性，避免指定可能超过 PD 当前分配的最大 TSO（参考 Grafana PD 面板上 `Current TSO`）的未来时间。否则，可能破坏并发处理线性一致性以及事务隔离级别，导致严重的数据正确性的问题。
+> 在执行 FLASHBACK 时，数据不会通过事务进行清理。请确保在 FLASHBACK 完成后，未来使用 TiDB 的任何历史版本读取功能（如 Stale Read 或 Set Snapshot）时，选择的时间点不要落在 FLASHBACK 运行的时间范围内。如果读取的历史版本包含未完成的 FLASHBACK 数据，可能破坏并发处理线性一致性以及事务隔离级别，导致严重的数据正确性的问题。
```
Comment (BornChanger) [inline sql-statements/sql-statement-flashback-cluster.md:16]:
```suggestion
> 在执行 FLASHBACK 时，数据清理过程中不能保证事务的一致性。请确保在 FLASHBACK 完成后，未来使用 TiDB 的任何历史版本读取功能（如 Stale Read 或 Set Snapshot）时，选择的时间点不要落在 FLASHBACK 运行的时间范围内。如果读取的历史版本包含未完成的 FLASHBACK 数据，可能破坏并发处理线性一致性以及事务隔离级别，导致严重的数据正确性的问题。
```

**Review by BornChanger — APPROVED** [review]

File: sql-statements/sql-statement-flashback-cluster.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ TiDB v6.5.6、v7.1.3、v7.5.1、v7.6.0 开始引入了 `FLASHBACK CLUSTER TO TSO
 > **警告：**
 >
 > 在指定恢复时间点时，请务必检查 TIMESTAMP 或 TSO 的有效性，避免指定可能超过 PD 当前分配的最大 TSO（参考 Grafana PD 面板上 `Current TSO`）的未来时间。否则，可能破坏并发处理线性一致性以及事务隔离级别，导致严重的数据正确性的问题。
+> 在执行 FLASHBACK 时，数据清理过程中不能保证事务的一致性。请确保在 FLASHBACK 完成后，未来使用 TiDB 的任何历史版本读取功能（如 Stale Read 或 Set Snapshot）时，选择的时间点不要落在 FLASHBACK 运行的时间范围内。如果读取的历史版本包含未完成的 FLASHBACK 数据，可能破坏并发处理线性一致性以及事务隔离级别，导致严重的数据正确性的问题。
```
Comment (qiancai) [inline sql-statements/sql-statement-flashback-cluster.md:16]:
```suggestion
> 在 `FLASHBACK CLUSTER` 执行期间，数据清理过程不能保证事务的一致性。在 `FLASHBACK CLUSTER` 执行完成后，如需使用 TiDB 的任何历史版本读取功能（如 [Stale Read](/stale-read.md) 或 [`tidb_snapshot`](/read-historical-data.md)），请确保所选择的历史时间点不在 FLASHBACK 运行的时间范围内。如果读取的历史版本包含未 FLASHBACK 完成的数据，可能会破坏并发处理的线性一致性以及事务隔离级别，导致严重的数据准确性问题。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: sql-statements/sql-statement-flashback-cluster.md [hunk 3]
```diff
@@ -12,7 +12,8 @@ TiDB v6.5.6、v7.1.3、v7.5.1、v7.6.0 开始引入了 `FLASHBACK CLUSTER TO TSO
 
 > **警告：**
 >
-> 在指定恢复时间点时，请务必检查 TIMESTAMP 或 TSO 的有效性，避免指定可能超过 PD 当前分配的最大 TSO（参考 Grafana PD 面板上 `Current TSO`）的未来时间。否则，可能破坏并发处理线性一致性以及事务隔离级别，导致严重的数据正确性的问题。
+> - 在指定恢复时间点时，请务必检查 TIMESTAMP 或 TSO 的有效性，避免指定可能超过 PD 当前分配的最大 TSO（参考 Grafana PD 面板上 `Current TSO`）的未来时间。否则，可能破坏并发处理线性一致性以及事务隔离级别，导致严重的数据正确性的问题。
+> - 在 `FLASHBACK CLUSTER` 执行期间，数据清理过程不能保证事务的一致性。在 `FLASHBACK CLUSTER` 执行完成后，如需使用 TiDB 的任何历史版本读取功能（如 [Stale Read](/stale-read.md) 或 [`tidb_snapshot`](/read-historical-data.md)），请确保所选择的历史时间点不在 FLASHBACK 运行的时间范围内。如果读取的历史版本包含未 FLASHBACK 完成的数据，可能会破坏并发处理的线性一致性以及事务隔离级别，导致严重的数据准确性问题。
```
Comment (qiancai) [inline sql-statements/sql-statement-flashback-cluster.md:16]:
```suggestion
> - 在 `FLASHBACK CLUSTER` 执行期间，数据清理过程不能保证事务的一致性。在 `FLASHBACK CLUSTER` 执行完成后，如需使用 TiDB 的任何历史版本读取功能（如 [Stale Read](/stale-read.md) 或 [`tidb_snapshot`](/read-historical-data.md)），请确保所选择的历史时间点不在 FLASHBACK 运行的时间范围内。如果读取的历史版本包含未 FLASHBACK 完成的数据，可能会破坏并发处理的线性一致性以及事务隔离级别，导致严重的数据正确性问题。
```

**Pull request merged**